pkgs/by-name/fo/font-alias/package.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  writeScript,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "font-alias";
+  version = "1.0.5";
+
+  src = fetchurl {
+    url = "mirror://xorg/individual/font/font-alias-${finalAttrs.version}.tar.xz";
+    hash = "sha256-n4niF7tz4ONjagpJP7+LfJlRVuDFPZoEdtIBtnwta24=";
+  };
+
+  passthru = {
+    updateScript = writeScript "update-${finalAttrs.pname}" ''
+      #!/usr/bin/env nix-shell
+      #!nix-shell -i bash -p common-updater-scripts
+
+      version="$(list-directory-versions --pname ${finalAttrs.pname} \
+        --url https://xorg.freedesktop.org/releases/individual/font/ \
+        | sort | tail -n1)"
```

Comment:
Is plain `sort` right, or do we want `sort -V`?

Reply:
ye we want `sort -V`